.github/workflows/full-test-jsonl.yml

```
@@ -31,80 +34,80 @@ jobs:
               if: always()
               run: |
-                  docker stop photon-test-pr-${{ github.event.pull_request.number }} || true
-                  docker rm photon-test-pr-${{ github.event.pull_request.number }} || true
-                  docker rmi photon-test:pr-${{ github.event.pull_request.number }} || true
+                docker stop photon-test-pr-${{ github.event.pull_request.number }} || true
+                docker rm photon-test-pr-${{ github.event.pull_request.number }} || true
+                docker rmi photon-test:pr-${{ github.event.pull_request.number }} || true
 
             - name: Output summary
               if: always()
               run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:37: Double quote to prevent globbing and word splitting

(shellcheck)